ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,115 @@
+
+## Introduction
+
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. We currently have one main HPC server, which handles all of our HPC jobs. However, have plans to expand the cluster to maximize the use of the resources at our disposal.
+
+## Gaining Access
+
+In order to access the HPC cluster, please send an access request to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or [group account][group] name and a detailed technical description of the projects you plan to run on our HPC infrastructure. This would include information about the nature of the software being run, as well as the amount of computational resources that are expected to be needed.
+
+### Connecting
+
+Our Slurm master node is named `hpcctl`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
+
+```
+ssh my_ocf_username@hpcctl.ocf.berkeley.edu
+```
+
+If you have trouble connecting please [contact us][contact] at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu), or come to [staff hours][staff_hours] when the lab is open and chat with us in person. We also have a hpc_users [slack][fco] channel where you can ask questions and talk to us about anything HPC.
+
+## The Cluster
+
+Users are currenly able to run jobs on our main HPC server, which has the following specifications:
+
+* 2x NVIDIA GTX 1080 Ti
+* 1x Intel Intel(R) Xeon(R) CPU E5-2640 v4 @2.40 GHz
+* 4x 16GB ECC DDR4-2400 RAM
+
+We have also recently been approved a grant that will be awarded in the coming weeks. So in addition to the above, the following will be installed in our HPC server:
+
+* 2x NVIDIA GTX 1080 Ti
+* 1x Intel Intel(R) Xeon(R) CPU E5-2640 v4 @2.40 GHz
+* 12x 16GB ECC DDR4-2400 RAM
+
+## Slurm
+
+We currently use [Slurm][slurm] as our workload manager for the cluster. We will soon post technical documentation about our Slurm configuration, but briefly, Slurm is a free and open-source job scheduler which helps distribute jobs from all users evenly among HPC computers, referred to as nodes. In short, all of your programs will be run through Slurm. To use Slurm there are several commands that will be helpful:
+
+* `srun`: Used to submit jobs.
+* `scontrol`: Used to view and edit the Slurm configuration.
+* `squeue`: Used to view running and queued jobs.
+
+## Dependencies
+For managing application dependencies, you currently have two options:
+
+### Virtual Environments
+
+First you could use a [virtual environment][venv] if you are using Python packages. To create a virtual environment navigate to your home directory and run the following commands:
+
+```
+virtualenv --system-site-packages -p python3 venv
```

Comment:
Are we sure we want to --system-site-packages? Shouldn't they want to isolate themselves from whatever we're running underneath, since lots of our code depends on our puppeted python debian packages and could possibly conflict with things they want to install?